cvat/apps/engine/management/commands/init_content_size.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from django.core.management.base import BaseCommand
+
+from cvat.apps.engine.models import Asset, Data
+from cvat.apps.engine.utils import get_paths_sizes
+
+
+class Command(BaseCommand):
+    help = "The command initializes 'content_size' value for 'Data' and 'Asset' objects"
+
+    def handle(self, *args, **options):
+        batch_size = 1000
+        for Model, get_directory_path, get_queryset in [
+            (Asset, lambda asset: asset.get_asset_dir(), lambda: Asset.objects),
+            (Data, lambda data: data.get_upload_dirname(), lambda: Data.objects.exclude(size=0)),
+        ]:
+            total = 0
+            self.stdout.write(f"Started for {Model.__name__}")
+            while True:
```

Comment:
```suggestion
            for db_objects in take_by(get_queryset().filter(content_size=None).iterator(), batch_size):
```

(`take_by` is in `cvat.apps.engine.utils`)